test/Exported.scala

```diff
@@ -1,11 +1,5 @@
 package scala.test
 
 object Exported {
-  def message: String = {
-    // terrible, don't do this in real code:
-    val msg = Class.forName("scala.test.Runtime")
```

Comment:
please don't remove this. This was testing something spefic: the presence of `scala.test.Runtime` on the runtime classpath. You should add what you need to make this work, not remove the testing of runtime classpaths.